services/gateway/apis/http-version3/swagger/definitions/transactions.json

```diff
@@ -655,42 +655,62 @@
 			"data": {
 				"type": "object",
 				"required": [
-					"transactionFeeEstimates",
-					"dynamicFeeEstimates"
+					"transaction"
 				],
 				"properties": {
-					"transactionFeeEstimates": {
+					"transaction": {
 						"type": "object",
 						"required": [
-							"minFee",
-							"accountInitializationFee"
+							"fee"
 						],
 						"properties": {
-							"minFee": {
-								"type": "string",
-								"example": "135000",
-								"description": "Minimum required transaction fee."
-							},
-							"accountInitializationFee": {
+							"fee": {
 								"type": "object",
 								"required": [
 									"tokenID",
-									"amount"
+									"minimum"
 								],
 								"properties": {
 									"tokenID": {
 										"type": "string",
-										"example": "0400000000000000",
+										"example": "0000000000000000",
 										"description": "The universal token identifier for the token."
 									},
-									"amount": {
+									"minimum": {
 										"type": "string",
 										"example": "5000000",
-										"description": "The extra fee to be supplied for initializing a new user account."
+										"description": "Minimum fee required for the transaction. This is the sum of the transaction byteFee and any other applicable additionalFee. The breakdown is available in the response meta."
+									},
+									"priority": {
+										"type": "object",
+										"required": [
+											"low",
+											"medium",
+											"high"
+										],
+										"description": "Transaction fee recommendations by priority based on the dynamic fees algorithm.",
+										"properties": {
+											"low": {
+												"type": "string",
+												"description": "Estimated transaction fee with low priority.",
+												"example": "135000"
+											},
+											"medium": {
+												"type": "string",
+												"description": "Estimated transaction fee with medium priority.",
+												"example": "207000"
+											},
+											"high": {
+												"type": "string",
+												"description": "Estimated transaction fee with high priority.",
+												"example": "405000"
+											}
+										}
 									}
+									
 								}
 							},
-							"messageFee": {
+							"params": {
```

Comment:
The `messageFee` object is missing within `data.transaction.params`. `tokenID` and `amount` must be the properties within `messageFee` here.